src/eval/mod.rs

```diff
@@ -12,11 +12,13 @@ mod dict;
 mod str;
 #[macro_use]
 mod value;
+
+pub mod ops;
```

Comment:
shouldn't be necessary to make it public due to `==` and `!=`

Reply:
Yup, nice observation. Thanks!